library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3205,6 +3205,16 @@
     <string name="tooltip_attachment_location">Share location</string>
     <string name="tooltip_attachment_voice_broadcast">Start a voice broadcast</string>
 
+    <string name="attachment_type_selector_gallery">Photo library</string>
+    <string name="attachment_type_selector_sticker">Stickers</string>
+    <string name="attachment_type_selector_file">Attachments</string>
+    <string name="attachment_type_selector_voice_broadcast">Voice broadcast</string>
+    <string name="attachment_type_selector_poll">Polls</string>
+    <string name="attachment_type_selector_location">Location</string>
+    <string name="attachment_type_selector_camera">Camera</string>
+    <string name="attachment_type_selector_contact">Contact</string>
+    <string name="attachment_type_selector_text_formatting">Text formatting</string>
```

Comment:
Lint is complaining that this is not used. If this is for future usage, you can ignore the error here, so that the string can get translated. Else please remove the string.

Reply:
I've removed it for now so that we don't forget to remove any lint ignore rule later

[`0650b6a`](https://github.com/vector-im/element-android/pull/7429/commits/0650b6ae22d0f0459e03ec6533d1fbde30b17203)